aibolit/patterns/var_middle/var_middle.py

```diff
@@ -119,6 +119,7 @@ def flatter(flat_children_list: List[Node], elem: Node) -> List[Node]:
             javalang.tree.BasicType,
             javalang.tree.CatchClauseParameter,
             javalang.tree.Annotation,
+            javalang.tree.TypeArgument,
```

Comment:
Why `TypeArgument` and not `LambdaExpression`?

Reply:
It is everything ok with LambdaExpression. But in the example I've created some templates. Their TypeArguments are spoiling the scope. So there was error on 12 (declaring lambda) and 13 (declaring optional) line, because of type parameters in variables types.
Maybe we have to instead of ignoring just nodes in AST, start ignoring whole subtrees. Just a plan for the future.

Reply:
okay, just asked.